ios/authentication.md

```diff
@@ -785,3 +819,402 @@ Note : `AWSGoogleSignIn` is only needed for using Google Login in your app and `
 	```
 
 Now, your drop-in UI will show a Google sign in button which the users can use to sign in to your app. This uses the `federatedSignIn()` flow underneath it.
+
+## Using Hosted UI for Authentication
```

Comment:
We need a description here of how this works, namely:
- How the Hosted UI is an OAuth flow that allows you to launch a login screen without embedding an SDK for Cognito or a social provider into your application
- How you can federate the social providers with the Hosted UI which exchanges the social tokens for a User Pool OIDC JWT token
- How you can also configure federation with Cognito Identity pools in the `awsconfiguration.json` and by doing so, when the user gets a User Pool token (even from a social provider) the SDK will automatically exchange this with Cognito Identity to also retrieve AWS credentials.

Reply:
Got it. Will elaborate.

Reply:
@undefobj 

Could you wordsmith this text and provide a starting point for what you want to be included in the preamble before we going to the specifics of setting up the Hosted UI?

Reply:
Note: The User Pool OIDC JWT token obtained from a successful sign-in will be federated into a configured Cognito Identity pool in the `awsconfiguration.json` and the SDK will automatically exchange this with Cognito Identity to also retrieve AWS credentials.